lib/src/scss-shared/_nx-focus-border-helpers.scss

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+// For dynamically calculating focus border styles
+// based on height, width and focus border offset
+// Calculations based on a 1px border
+
+// NxRadio and NxCheckbox require the left property for correct positioning of focus border
+@mixin nx-radio-checkbox-focus-border($height, $width, $offset) {
```

Comment:
It's not super clear to me what the `$offset` represents.  It seems to be the total extra space between the control and the focus border on each axis? That seems a little confusing.  Additionally we are using the same value everywhere for it so far so I'd be inclined not to provide a parameter for it at all.

Reply:
> It's not super clear to me what the `$offset` represents. It seems to be the total extra space between the control and the focus border on each axis? That seems a little confusing. Additionally we are using the same value everywhere for it so far so I'd be inclined not to provide a parameter for it at all.

You are correct. It is the total space between the border and the focus border in each axis. I initially put $offset as a variable to provide support for custom spacing, but looking back, I doubt that there will ever be a case when we would need custom spacing for something like this. So this has been reverted to have fixed spacing and the parameter has been removed.
fa154a89ff2754976fbd5640f59e3fdb15d88671